puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/scm/foreman.yaml

```diff
@@ -2,7 +2,8 @@
     name: foreman
     scm:
       - git:
-         url: git://github.com/theforeman/foreman.git
-         wipe-workspace: true
-         branches:
-          - 'develop'
+          url: git://github.com/theforeman/foreman.git
+          wipe-workspace: true
+          basedir: foreman
```

Comment:
Reverted that change for now, as it applied for the case of using multiple scm and that will be done as a re-factor after this.